docs/src/components/MDXContent/renderers.js

```diff
@@ -0,0 +1,157 @@
+
+/* @flow strict */
+
+import React from 'react';
+import { cssModules } from 'bpk-react-utils';
+import {
+  BpkTable,
+  BpkTableHead,
+  BpkTableBody,
+  BpkTableRow,
+  BpkTableCell,
+  BpkTableHeadCell,
+} from 'bpk-component-table';
+import BpkText from 'bpk-component-text';
+import { BpkCodeBlock } from 'bpk-component-code';
+import BpkMobileScrollContainer from 'bpk-component-mobile-scroll-container';
+import BpkLink from 'bpk-component-link';
+import BpkBlockquote from 'bpk-component-blockquote';
+
+import Paragraph from '../Paragraph';
+
+import STYLES from './MDXContent.scss';
+
+const getClassName = cssModules(STYLES);
+
+const TAG_NAMES = ['h1', 'h2', 'h3', 'h4', 'h5'];
+const TEXT_STYLES = ['xxl', 'xl', 'lg', 'base', 'sm'];
+
+type Props = {
+  darkBackground: boolean,
+};
+
+const Renderer = (props: Props) => {
+  const { darkBackground } = props;
+  const renderers = {};
+
+  renderers.a = BpkLink;
+  renderers.blockquote = BpkBlockquote;
+
+  renderers.code = (codeBlockProps: { value: string }) => {
+    const { value, ...codeBlockRest } = codeBlockProps;
+    return <BpkCodeBlock {...codeBlockRest}>{value}</BpkCodeBlock>;
+  };
+
+  renderers.pre = (codeBlockProps: { value: string }) => {
+    const { value, ...codeBlockRest } = codeBlockProps;
+    return <BpkCodeBlock {...codeBlockRest}>{value}</BpkCodeBlock>;
+  };
+
+  TAG_NAMES.forEach(tag => {
+    renderers[tag] = (headingProps: { children: string }) => {
+      const { children, ...headingRest } = headingProps;
+
+      const flatten = (text, child) => {
+        return typeof child === 'string'
+          ? text + child
+          : React.Children.toArray(child.props.children).reduce(flatten, text);
+      };
+
+      // https://github.com/remarkjs/react-markdown/issues/69#issuecomment-289860367
+      const id = React.Children.toArray(children)
+        .reduce(flatten, '')
+        .toLowerCase()
+        .replace(/\W/g, '-');
+
+      return (
+        // $FlowFixMe[cannot-spread-inexact] - inexact rest. See 'https://github.com/Skyscanner/backpack/blob/master/decisions/flowfixme.md'.
+        <BpkText
+          id={id}
+          textStyle={TEXT_STYLES[TAG_NAMES.indexOf(tag)]}
+          tagName={tag}
+          className={getClassName(`bpkdocs-mdx-page__heading-${tag}`)}
+          {...headingRest}
+        >
+          {children}
+        </BpkText>
+      );
+    };
+  });
+
+  renderers.img = (imageProps: { alt: string, src: string }) => {
+    const { alt, src, ...imageRest } = imageProps;
+    return (
+      // Can't use BpkImage here because it requires us to know width and height.
+      // $FlowFixMe[cannot-spread-inexact] - inexact rest. See 'https://github.com/Skyscanner/backpack/blob/master/decisions/flowfixme.md'.
+      <img /* eslint-disable-line backpack/use-components */
+        alt={alt}
+        src={src}
+        className={getClassName('bpkdocs-mdx-page__image')}
+        {...imageRest}
+      />
+    );
+  };
+
+  renderers.link = BpkLink;
```

Comment:
It doesn't seem like this line will do anything — you're already accounting for `a` elements earlier in this file and according to https://mdxjs.com/getting-started#table-of-components, `link` isn't a valid tag.

Reply:
Ah good spot missed this when migrating